lib/Core/Executor.cpp

```diff
@@ -1331,14 +1331,23 @@ void Executor::executeCall(ExecutionState &state, KInstruction *ki, Function *f,
     case Intrinsic::not_intrinsic: {
       // Here we replace cell vector with expression vector. We assume they are
       // short.
-      std::vector<ref<Expr> > exprArguments;
+      /*std::vector<ref<Expr> > exprArguments;
       for (std::vector<Cell>::iterator it = arguments.begin(),
                                        ie = arguments.end();
            it != ie; ++it) {
         exprArguments.push_back(it->value);
       }
+
+      //
+      std::vector<ref<Expr> > exprArgumentsError;
+                        for (std::vector<Cell>::iterator it = arguments.begin(),
+      ie =
+                                        arguments.end(); it != ie; ++it) {
+                                exprArgumentsError.push_back(it->error);
+                        }*/
+
```

Comment:
@Himeshi I recommend deleting the commented code here for readability. Because of version control such as git, we can always checkout old version.